tests/test_layers.py

```diff
@@ -33,3 +33,44 @@ def test_sizes(self):
 
         for surface in self.surface_layers:
             assert surface.get_size() == self.TARGET_SURFACE_SIZE
+
+    def test_render(self):
+
+        subsurface_size = (self.TARGET_SURFACE_SIZE[0] / 2,
+                           self.TARGET_SURFACE_SIZE[1] / 2)
+
+        # Create our test surfaces
+        background = pygame.surface.Surface(self.TARGET_SURFACE_SIZE)
+        top_left = pygame.surface.Surface(subsurface_size)
+        top_right = pygame.surface.Surface(subsurface_size)
+        bottom_left = pygame.surface.Surface(subsurface_size)
+
+        # Fill the surfaces
+        background.fill((255, 255, 255))
+        top_left.fill((255, 0, 0))
+        top_right.fill((0, 255, 0))
+        bottom_left.fill((0, 0, 255))
+
+        # Create a surface to compare with and blit our test surfaces
+        test_surface = pygame.surface.Surface(self.TARGET_SURFACE_SIZE)
+        test_surface.blit(background, (0, 0))
+        test_surface.blit(top_left, (0, 0))
+        test_surface.blit(top_right, (subsurface_size[0], 0))
+        test_surface.blit(bottom_left, (0, subsurface_size[1]))
+
+        # Create the SurfaceLayers object and fill it with our layers
+        surface_layers = sappho.SurfaceLayers(self.target_surface, 4)
+        surface_layers[0].blit(background, (0, 0))
+        surface_layers[1].blit(top_left, (0, 0))
+        surface_layers[2].blit(top_right, (subsurface_size[0], 0))
+        surface_layers[3].blit(bottom_left, (0, subsurface_size[1]))
```

Comment:
Maybe it'd be better to have the first layer be a canvas be this color instead of filling in the bottom right quarter?